### PR TITLE
[TTAHUB-2698] src/tools, coverage >= 90%

### PR DESCRIPTION
**Pull request by nvms**
## Description of change

Increase coverage for src/tools to 90%.

```
-------------------------------------|---------|----------|---------|---------|-------------------------------------
File                                 | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                   
-------------------------------------|---------|----------|---------|---------|-------------------------------------
All files                            |   97.48 |    90.22 |     100 |   97.51 |                                     
 bootstrapAdmin.js                   |     100 |      100 |     100 |     100 |                                     
 changeReportStatus.js               |     100 |      100 |     100 |     100 |                                     
 dataValidation.js                   |     100 |      100 |     100 |     100 |                                     
 goalPilot.js                        |     100 |      100 |     100 |     100 |                                     
 importPlanGoals.js                  |   87.34 |     92.3 |     100 |   87.34 | 146-147,155-162,195                 
 importSystem.ts                     |     100 |      100 |     100 |     100 |                                     
 logicalDataModel.ts                 |    97.8 |    94.35 |     100 |   98.11 | 154,160,487-488                     
 populateLegacyResourceTitles.js     |     100 |      100 |     100 |     100 |                                     
 processData.js                      |     100 |       75 |     100 |     100 | 105,110,128-130,179,198-291,384-440 
 updateCompletedEventReportPilots.js |     100 |      100 |     100 |     100 |                                     
-------------------------------------|---------|----------|---------|---------|-------------------------------------
```

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2698

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/tools/logicalDataModel.test.js [hunk 1]
```diff
@@ -39,4 +53,351 @@ describe('logicalDataModel', () => {
...
+        { model: { name: 'Goal' }, table: 'Goal', attributes: [] },
+      ];
+
+      const result = processAssociations(associations, tables, schemas);
+
+      expect(result).toContain('!issue=\'associations need to be defined both directions\'');
+      expect(result).toContain('Goal "n" }--[#d54309,dotted,thickness=2]--{ "n" Grant : enrollments');
+    });
+
+    // eslint-disable-next-line jest/no-commented-out-tests
```
Comment (GarrettEHill) [inline src/tools/logicalDataModel.test.js:332]: Please delete bulk comments

Comment (nvms) [reply src/tools/logicalDataModel.test.js]: 👍 fe240f4f002b8ed9b5caf07cae0e6978fe4eeef1

File: src/models/objectiveTemplate.js [hunk 2]
```diff
@@ -14,12 +14,12 @@ export default (sequelize, DataTypes) => {
...
-        foreignKey: 'objectiveTemplateId',
-        otherKey: 'goalTemplateId',
-        as: 'goalTemplates',
-      });
+      // ObjectiveTemplate.belongsToMany(models.GoalTemplate, {
+      //   through: models.GoalTemplateObjectiveTemplate,
+      //   foreignKey: 'objectiveTemplateId',
+      //   otherKey: 'goalTemplateId',
+      //   as: 'goalTemplates',
+      // });
```
Comment (thewatermethod) [inline src/models/objectiveTemplate.js:22]:
```suggestion
      // ObjectiveTemplate.belongsToMany(models.GoalTemplate, {
      //   through: models.GoalTemplateObjectiveTemplate,
      //   foreignKey: 'objectiveTemplateId',
      //   otherKey: 'goalTemplateId',
      //   as: 'goalTemplates',
      // });
```
```suggestion 
```

**Review by thewatermethod — APPROVED** [review]

File: src/models/objectiveTemplate.js (hunk 2 above)
Comment (thewatermethod) [reply src/models/objectiveTemplate.js:22]: not sure what I did with the suggestions there, but the idea was deleting the commented out association

**Review by kryswisnaskas — APPROVED** [review]
Very nice! Thank you.

**Pull request merged**